config/build.ts

```diff
@@ -157,7 +160,10 @@ async function processSnippets(dir, processedSnippets) {
                 order: (typeof (snippet as any).order === 'undefined') ? 100 /* nominally 100 */ : (snippet as any).order,
                 api_set: snippet.api_set
             };
-
+            if (isPrivateSample) {
```

Comment:
Seems like we can set this directly in returned object:
return {
 id: snippet.id,
type: isPrivateSample? 'private' : 'public',
name: snippet.name